portlet-hibernate-util/src/main/java/org/jasig/portlet/utils/hibernate/dialect/Oracle10gDialect.java

```diff
@@ -0,0 +1,20 @@
+package org.jasig.portlet.utils.hibernate.dialect;
+
+import java.sql.Types;
+
+/**
+ * A derived dialect of Oracle10gDialect that uses clobs over Oracle LONG.
+ */
+public class Oracle10gDialect extends org.hibernate.dialect.Oracle10gDialect {
```

Comment:
I think we should use a different/modified name for this class -- something like `ClobOracle10gDialect`.

Why does it use "clobs over Oracle LONG?"  In what circumstances?